examples/src/main/java/com/mapbox/navigation/examples/core/MapboxBuildingHighlightActivity.kt

```diff
@@ -358,68 +290,31 @@ class MapboxBuildingHighlightActivity : AppCompatActivity(), OnMapLongClickListe
         init()
     }
 
-    /**
-     * Show all buildings on the map.
-     */
-    private fun extrudeBuildings(extrudeBuildings: Boolean) {
-        this.extrudeBuildings = extrudeBuildings
-        updateFillBuildingsLayer(buildingHighlightApi.highlightedBuildings)
-    }
-
     override fun onStart() {
         super.onStart()
-        binding.mapView.onStart()
```

Comment:
Does the map handle the lifecycle events internally now?

Reply:
Yes. I get a warning saying we don't need these anymore. That's great!!

Reply:
![Screen Shot 2021-08-17 at 12 38 16 PM](https://user-images.githubusercontent.com/9770186/129789630-b79e0a6e-a2f6-4153-a1fa-4c3cc0653363.png)